Cargo.toml

```diff
@@ -34,5 +34,15 @@ incremental = false
 [workspace.lints.rust]
 missing_debug_implementations = "warn"
 
+# We use this --cfg for documenting the cargo features on which an API
+# is available.  To preview this locally use: RUSTFLAGS="--cfg
+# iroh_docsrs cargo +nightly doc --all-features".  We use our own
+# iroh_docsrs instead of the common docsrs to avoid also enabling this
+# feature in any dependencies, because some indirect dependencies
+# require a feature enabled when using `--cfg docsrs` which we can not
+# do.  To enable for a crate set `#![cfg_attr(iroh_docsrs,
+# feature(doc_cfg))]` in the crate.
+unexpected_cfgs = { level = "warn", check-cfg = ["cfg(iroh_docsrs)"] }
```

Comment:
don't we need to specify this feature in the individual `Cargo.toml`s?

Reply:
no, it's not even allowed: they all have `lints.workspace = true` so pick it up from here.

Reply:
I see, that's complicated